From f5a7a70896708c35a25e09651fb69652e5aeeab2 Mon Sep 17 00:00:00 2001
From: Victor Burton <vburton@z3r6p4.42lyon.fr>
Date: Wed, 7 Jan 2026 10:38:29 +0100
Subject: [PATCH] kbuild: change EXTRAVERSION to -thor_kernel

Signed-off-by: Victor Burton <vburton@z3r6p4.42lyon.fr>
---
 Assignment02/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Assignment02/Makefile b/Assignment02/Makefile
index 665b79a..f2b6274 100644
--- a/Assignment02/Makefile
+++ b/Assignment02/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 19
 SUBLEVEL = 0
-EXTRAVERSION = -rc4
+EXTRAVERSION = -thor_kernel
 NAME = Baby Opossum Posse
 
 # *DOCUMENTATION*
-- 
2.34.1

